src/components/post-comments.jsx

```diff
@@ -57,8 +57,12 @@ export default (props) => {
       props.toggleCommenting(props.post.id)
     }
 
-    const updatedCommentText = `@${username} ` + (props.post.newCommentText || '')
-    props.updateCommentingText(props.post.id, updatedCommentText)
+    const text = (props.post.newCommentText || '')
+    const prefix = `@${username}`
+
+    if (text.indexOf(prefix) === -1){
+      props.updateCommentingText(props.post.id, `${prefix} ${text}`)
+    }
```

Comment:
It's about 'don't add the same username for the second time', but more like 'don't add two same usernames in a row' seems much more legit